tasks/views.py

```diff
@@ -1,3 +1,33 @@
 # from django.shortcuts import render
+from django.shortcuts import redirect, render
+from tasks.forms import TaskForm
+from tasks.models import Task
+from django.contrib import messages
+
 
 # Create your views here.
+def new_task(request):
+    if request.method == "POST":
+        form = TaskForm(request.POST)
+        if form.is_valid():
+            taskForm = form.save(commit=False)
+            try:
+                Task.create_task(taskForm.title, taskForm.assignee, taskForm.created_by, taskForm.priority,
+                                 taskForm.status,
+                                 taskForm.description)
```

Comment:
I think you can write it in the same line as you did it in the line above.
please be consistent in your code.
```suggestion
                                 taskForm.status, taskForm.description)
```